data/random-teams.ts

```diff
@@ -1348,7 +1343,8 @@ export class RandomTeams {
 			item = 'Leftovers';
 
 		// Better than Leftovers
-		} else if (isLead && !['Disguise', 'Sturdy'].includes(ability) && !hasMove['substitute'] && !counter['recoil'] && !counter['recovery'] && species.baseStats.hp + species.baseStats.def + species.baseStats.spd < 255 && !isDoubles) {
+		} else if (isLead && !['Disguise', 'Sturdy'].includes(ability) && !hasMove['substitute'] && !hasMove['tailslap'] && !counter['recoil'] && !counter['recovery'] &&
+			species.baseStats.hp + species.baseStats.def + species.baseStats.spd < 255 && !isDoubles) {
```

Comment:
I don't think Sash Cinccino should be removed.

Reply:
At least not with a really specific hardcode like this.

Reply:
i mean, you already hardcoded it to give it Life Orb

Reply:
and it's reported at least once a week. But you can remove it if you want.